migrate.py

```diff
@@ -0,0 +1,77 @@
+import os
+import psycopg2
+from config import dbconfig, filename, section
+
+
+
+def create_tables():
+    """create tables in postgresql database"""
+    commands = (
+        """
+        DROP TABLE IF EXISTS tb_users;
+        CREATE TABLE tb_users(
+            user_id SERIAL PRIMARY KEY,
+            username VARCHAR(100) UNIQUE NOT NULL,
+            firstname VARCHAR(50) NOT NULL,
+            lastname VARCHAR(50) NOT NULL,
+            password VARCHAR(255) NOT NULL,
+            created_on TIMESTAMP NOT NULL,
+            last_login TIMESTAMP
+        )
+        """
+        ,
+        """
+        DROP TABLE IF EXISTS tb_request;
+        CREATE TABLE tb_request(
+            request_id SERIAL PRIMARY KEY,
+            requestor VARCHAR(255) NOT NULL,
+            email VARCHAR(100) NOT NULL,
+            type VARCHAR(50) NOT NULL,
+            status VARCHAR(50) NOT NULL,
+            description TEXT,
+            created_on TIMESTAMP NOT NULL
+        )
+        """
+        ,
+        """
+        DROP TABLE IF EXISTS tb_admin;
```

Comment:
I don't think you require a new table to store the admin data, this could just be in the users table.

Reply:
Cool, I'll remove this.